app/views/admin/booths/new.html.haml

```diff
@@ -1,3 +1,3 @@
 %h1 New #{t'booth'}
 
-= render 'booths/form'
+= render 'booths/form', conference: @conference
```

Comment:
Why do we need to pass the conference variable and not use @conference inside the form?

Reply:
@differentreality Actually [haml-lint](https://github.com/sds/haml-lint) keeps complaining about using instance variable in partials. This has been issue in my other pr's too, maybe I should go ahead and generate a to-do file for all these changes?